File: 02. Tech Module/02. Software Techologies/18. Exam Prep - 3/LogNoziroh/JavaScript-Skeleton/models/Report.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    //TODO
    let Report = sequelize.define('Report', {
        status: {
            type: Sequelize.TEXT,
            values:["Normal",'Warning', 'Critical'],
            allowNull: false,
        },
        message:{
            type: Sequelize.TEXT,
            allowNull: false,
        },
        origin:{
            type: Sequelize.TEXT,
            allowNull: false,
        }
    },{
        timestamps:false
    });

    return Report;

};

File: 02. Tech Module/02. Software Techologies/18. Exam Prep - 3/LogNoziroh/CSharp-Skeleton/LogNoziroh/Models/Report.cs
namespace LogNoziroh.Models
{
    using System.ComponentModel.DataAnnotations;

    public class Report
    {
        //TODO
        [Key]
        public int Id { get; set; }

        [Required]
        public string Status { get; set; }

        [Required]
        public string Message { get; set; }

        [Required]
        public string Origin { get; set; }

    }
}
